puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/build/build_rpm_rubygem-katello_nightly.yaml

```diff
@@ -0,0 +1,32 @@
+- job:
+    name: build_rpm_rubygem-katello_nightly
+    node: fast
+    properties:
+      - tfm-build-discarder
+    triggers:
+      - github
+    scm:
+      - git:
+          url: https://github.com/Katello/katello.git
+          wipe-workspace: true
+          branches:
+            - master
+    wrappers:
+      - abort_build:
+          timeout_min: 60
+      - ansicolor
+    builders:
+      - shell: !include-raw: scripts/build_rpm_rubygem-katello_nightly.sh
+    publishers:
+      - archive:
+          artifacts: 'pkg/*'
+      - trigger-parameterized-builds:
```

Comment:
With the change to this being a publisher, this job will now look like it succeed if all it did was generate the gem successfully which I think is confusing based upon the name of the job. This is partially the reason that this was a builder before and not a publisher.

Reply:
I set it as a publisher due to how `packaging_build_rpm` looks for the `lastSuccessfulBuild` of a referenced job to get the source. I do understand that a more accurate name could mitigate confusion.

Reply:
You can keep this as a builder but adopting the same strategy used in the newer hammer-cli and katello-installer build jobs. Take a look at:

https://github.com/theforeman/foreman-infra/blob/master/puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/hammerCliKatelloNightlyRelease.groovy#L24-L30

Reply:
updated!